runtime/lua/vim/_matchpairs.lua

```diff
@@ -20,6 +20,13 @@ local function jump_to_node(node, start)
   api.nvim_win_set_cursor(0, { row + 1, col })
 end
 
+--- @param node TSNode
+local function anonymous_children(node)
```

Comment:
Note for later: consider `vim._,memoize` for potentially expensive treesitter calculations.